web/src/ui_init.js

```diff
@@ -714,7 +714,9 @@ export function initialize_everything() {
     linkifiers.initialize(page_params.realm_linkifiers);
     realm_playground.initialize(page_params.realm_playgrounds, generated_pygments_data);
     compose.initialize();
-    composebox_typeahead.initialize(); // Must happen after compose.initialize()
+    composebox_typeahead.initialize({
+        on_enter_send: compose.finish,
+    }); // Must happen after compose.initialize()
```

Comment:
As a general rule we prefer to put comments on their own line by themselves, and probably we should just move this one here.